{"name": "Проверка принятия заказа, используя невалидный id заказа", "status": "passed", "steps": [{"name": "Попытка принять заказ, используя невалидный id заказа", "status": "passed", "steps": [{"name": "Авторизация курьера в системе", "status": "passed", "steps": [{"name": "Регистрация нового курьера", "status": "passed", "start": 1736714121262, "stop": 1736714121617}], "start": 1736714121262, "stop": 1736714121941}], "start": 1736714121262, "stop": 1736714122256}], "start": 1736714121262, "stop": 1736714122256, "uuid": "4f9287e3-1334-4b17-b71d-285dbf9e60fb", "historyId": "40bc7fc711221da5939448f7f9340e3e", "testCaseId": "40bc7fc711221da5939448f7f9340e3e", "fullName": "tests.orders.test_take_order.TestTakeOrder#test_take_order_with_wrong_order_id", "labels": [{"name": "parentSuite", "value": "tests.orders"}, {"name": "suite", "value": "test_take_order"}, {"name": "subSuite", "value": "TestTakeOrder"}, {"name": "host", "value": "MacBook-Pro-Alina.local"}, {"name": "thread", "value": "12315-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.orders.test_take_order"}]}